libkbfs/crypto_common.go

```diff
@@ -294,15 +294,15 @@ func (c CryptoCommon) DecryptPrivateMetadata(
 
 const minBlockSize = 256
 
-// nextPowerOfTwo returns next power of 2 greater than the input n.
+// thisPowerOfTwo returns next power of 2 greater than or equal to the input n.
 // https://en.wikipedia.org/wiki/Power_of_two#Algorithm_to_round_up_to_power_of_two
-func nextPowerOfTwo(n uint32) uint32 {
-	if n < minBlockSize {
+func thisPowerOfTwo(n int) int {
```

Comment:
@patrickxb: do you remember why you originally implemented this function as `nextPowerOfTwo` (called when doing KBFS block padding)?  I can't remember if we had a good reason for it or not.